packages/apps/crm/src/api/event-api.js

```diff
@@ -139,3 +140,43 @@ export async function getEvent({eventId}) {
     fieldKey: 'crm_event',
   });
 }
+
+export async function createEvent({event}) {
+  return axiosApiProvider.put({
+    url: '/ws/rest/com.axelor.apps.crm.db.Event',
+    data: {
+      data: event,
+    },
+  });
+}
+
+export async function updateEvent({event}) {
+  let dataToSend = event;
```

Comment:
We could just use event, no need to do this decomposition. But it seems to doesn't work without doing that.